include/libtorrent/alert_types.hpp

```diff
@@ -2064,6 +2065,8 @@ namespace libtorrent
 		boost::array<char, 64> signature;
 		std::string salt;
 		boost::uint64_t seq;
+
+		bool success;
```

Comment:
Will add comments for sure. actually I also not sure when we should set it as false.

I added some logs for testing, what I saw before is, sometimes some node response a error like "invalid token", sometimes is "unknown request" (old version I guess?), and if I use a wrong private key, all nodes reply "invalid signature".

Right now, only when (success response >= error responses), I set success to true, but i'm not sure if it's the best choice here.  (BTW, there is no another put later)